### PR TITLE
Update docs about caches, specifically hazelcast

### PR DESCRIPTION
**Pull request by Primer42**
Documenting https://github.com/tumblr/collins/pull/367

@tumblr/systems

### CODE REVIEW

File: _includes/configuration/cache.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+<div class="example reference-config">
+    <pre>cache {
+  enabled=true
+
+  type=in-memory
+}</pre>
+</div>
+
+<h3>Guava in-memory cache</h3>
+Guava cache is only sutiable for standalone setup.  It is used to cache various information internally to collins.
```
Comment (defect) [inline _includes/configuration/cache.html]: suitable

File: _includes/configuration/cache.html [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    <thead>
+        <tr>
+            <th>Key</th><th>Type</th><th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td class="inlinecode">enabled</td>
+            <td>Boolean</td>
+            <td> Whether to enable the guava. Defaults to <span class="inlinecode">true</span>.  If you choose <span class="inlinecode">in-memory</span> for your cache type, you must set this configuration to <span class="inlinecode">true</span> as well.</td>
```
Comment (defect) [inline _includes/configuration/cache.html]: "Whether to enable the guava _cache_" perhaps?

File: _includes/configuration/cache.html [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    <thead>
+        <tr>
+            <th>Key</th><th>Type</th><th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td class="inlinecode">enabled</td>
+            <td>Boolean</td>
+            <td> Whether to enable the hazelcast. Defaults to <span class="inlinecode">true</span>.  If you choose <span class="inlinecode">distributed</span> for your cache type, yuu must set this configuration to <span class="inlinecode">true</span> as well.</td>
```
Comment (defect) [inline _includes/configuration/cache.html]: s/yuu/you/

**Comment by Primer42** [comment]
Fixed all the spelling mistakes that @defect pointed out.

**Comment by qx-xp** [comment]
+1

**Comment by defect** [comment]
LGTM

**Pull request merged**